lib/task/import/csvCheckImportTask.class.php

```diff
@@ -71,14 +71,15 @@ protected function configure()
                 sfCommandOption::PARAMETER_OPTIONAL, 'The application name', 'qubit'
             ),
             new sfCommandOption('env', null, sfCommandOption::PARAMETER_REQUIRED,
```

Comment:
Although it takes a lot of vertical space, I think it's better to readability and consistency to put each argument on it's own line when they don't fit on a single line, e.g.

```
new sfCommandOption(
    'env', 
    null,
    sfCommandOption::PARAMETER_REQUIRED,
    'The environment', 
    'cli'
),
```

Reply:
Sounds good. It does read easier - change completed.